Cargo.lock

```diff
@@ -1776,7 +1776,7 @@ version = "0.8.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "bef552e6f588e446098f6ba40d89ac146c8c7b64aade83c051ee00bb5d2bc18d"
 dependencies = [
- "uuid 1.5.0",
```

Comment:
It would be nice if you revert this file=) But it is optional

Reply:
f4203c8b